src/isar/services/service_connections/mqtt/robot_status_publisher.py

```diff
@@ -37,6 +37,8 @@ def _get_combined_robot_status(
             return RobotStatus.Offline
         elif current_state == States.Idle and robot_status == RobotStatus.Available:
             return RobotStatus.Available
+        elif current_state != States.Idle or robot_status == RobotStatus.Stuck:
```

Comment:
Should this not be and instead of or?

Reply:
Think we only want one catch all on the bottom, but keeping the specific once with and

Reply:
I think we should decide this regardless of current state and make a decision based on robot_status only.